examples/md-flexible/scripts/generate-mdflex-configs.py

```diff
@@ -1,5 +1,30 @@
 #!/usr/bin/python3
+
+"""Generates a zoo of configurations for md-flexible
+
+The script generates hundreds of yaml configuration files,
+that aim to cover a wide range of possible simulation states.
+
+It is not simply the cross product of the range of parameters below,
+but also tries to exclude unfeasible combinations, e.g. huge particle
+numbers in tiny domains.
```

Comment:
See my comment on line 202 for my reasoning for this:

```suggestion
It is not simply the cross product of the range of parameters below,
but also tries to exclude unfeasible combinations, e.g. huge particle
numbers in tiny domains.

These exclusion criteria may not be appropriate for your test cases.
E.g. they assume a sigma of no smaller than one and a cutoff of no 
larger than 3.
```
(Potentially replace the assumption with whatever is more appropriate)